LabExT/View/Movement/MovementWizard.py

```diff
@@ -526,3 +565,381 @@ def _build_entry_with_label(
+        ----------
+        calibration: Calibration
+            Instance of a calibration
+        chip_axis: Axis
+            Requested chip axis to wiggle
+        """
+        if not self._confirm_wiggle(chip_axis):
+            return
+
+        try:
+            run_with_wait_window(
+                self.wizard,
+                f"Wiggle {chip_axis} of {calibration}",
+                lambda: calibration.wiggle_axis(chip_axis))
+        except RuntimeError as e:
+            messagebox.showerror(
+                "Error"
+                f"Wiggling {chip_axis} failed: {e}",
+                parent=self.wizard)
+
+    def _confirm_wiggle(self, axis) -> bool:
+        """
+        Confirms with user if wiggeling is allowed.
+        """
+        message = 'By proceeding this button will move the stage along the {} direction. \n\n'.format(axis) \
+                  + 'Please make sure it has enough travel range(+-5mm) to avoid collision. \n\n' \
+                  + 'For correct operation the stage should: \n' \
+                  + 'First: Move in positive {}-Chip-Axis direction \n'.format(axis) \
+                  + 'Second: Move in negative {}-Chip-Axis direction \n\n'.format(axis) \
+                  + 'If not, please check your assignments.\n Do you want to proceed with wiggling?'
+
+        return messagebox.askokcancel("Warning", message, parent=self.wizard)
+
+
+class CoordinatePairingStep(Step):
+    """
+    Wizard Step to fully calibrate stages.
+    """
+
+    def __init__(self, wizard, mover, chip) -> None:
+        """
+        Constructor for new Wizard step for fully calibrate stages.
+
+        Parameters
+        ----------
+        master : Tk
+            Tk instance of the master toplevel
+        mover : Mover
+            Instance of the current mover.
+        """
+        super().__init__(
+            wizard,
+            self.build,
+            finish_step_enabled=True,
+            on_reload=self.on_reload,
+            title="Stage Configuration")
+        self.mover: Type[MoverNew] = mover
+        self.chip: Type[Chip] = chip
+
+        self._use_input_stage_var = BooleanVar(self.wizard, True)
+        self._use_output_stage_var = BooleanVar(self.wizard, True)
+        self._full_calibration_new_pairing_button = None
+
+        self._coordinate_pairing_window = None
+
+        self.pairings = []
+
+    def build(self, frame: Type[CustomFrame]):
+        """
+        Builds step to fully calibrate axes.
+
+        Parameters
+        ----------
+        frame : CustomFrame
+            Instance of a customized Tkinter frame.
+        """
+        frame.title = "Calibrate Stage to enable absolute movement"
+
+        Label(
+            frame,
+            text="To move the stages absolutely in chip coordinates, define at least 3 stage-chip-coordinate pairings to calculate the rotation. \n" +
+            "Note: After one coordinate pairing TODO. Therefore this is only an approximation.").pack(
```

Comment:
what does this comment mean?

Reply:
Oops! That was a hint for me to add a description. I will change it. 😄

Reply:
Fixed!